iron-list.html

```diff
@@ -341,6 +341,13 @@
       return this._viewportSize * 3;
     },
 
+   /**
+    * True if the current list is visible.
+    */
+    get _isVisible() {
+      return this.offsetParent !== null;

```

Comment:
Sad... Unfortunately http://jsbin.com/vudinulutu/edit?html,console,output prints `true, true` for IE, meaning `offsetParent` isn't a reliable detector; sorry about that.

Probably just checking `offsetHeight` is sufficient, right?

Otherwise, LGTM